dbms/src/IO/ReadBuffer.h

```diff
@@ -59,6 +59,8 @@ class ReadBuffer : public BufferBase
         return res;
     }
 
+    /// Required to support buffers from infinite streams.
```

Comment:
"infinite streams" - not clear enough.

No explanation about what it does and what is the return value.